data/examples/71--custom_fonts.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Custom Fonts",
+  "toc": "on",
+  "tags": [
+    "CSS",
+    "Custom Fonts"
+  ],
+  "layout": "dock-left",
+  "key": 71,
+  "code": "#code/eJyNkE1qwzAQRvc+xaBFaCixSJeO7ZYU2gO0F1DksSWQLeFRKKLk7pWUPxe6KGgzaL73PaYmHwy2BcCLHp2dPRxn88CU944qzns7eSoHaweDwmkqpR25JHp67sWoTWj2x2lAfPxQosNVp8kZERr6Eo6td0WiJsKmFxLhO44AlzmFK2AyHHDuwoRsl39pllU2iB3ncn5bKb3v12ntlMDlITdvKDX/yT6rQVZj99yN9w+hU1Hzy4GKWm1BGkHUsGU1az+VJojPK4T3fCl4i0yQwhjsYKlRc7Vtl6R73S+MsTGcza6U1+tiRvwAmrCP3A==",
+  "info": [
+    {
+      "title": "Custom Fonts in CSS",
+      "focus": null,
+      "text": "This example show show to use the <code>@import</code> and <code>@font-face</code> CSS rules to use custom fonts on your website from both a remote file and local file."
```

Comment:
"show" is written twice here, should probably read "This example shows how to use..."